### ISSUE TITLE
Adding network speed in the bar

### ISSUE THREAD

**Issue by dajmo** [miklhh/i3blocks-config#3]
This i3blocks bar is awesome , I wonder if network-speed meter can be added

my bar: https://ibb.co/PjSp1R5

**Comment by miklhh** [comment]
Thank you for your kind words! The image in the link you posted is no longer available. Could you send a new link? I would love to see your bar.

**Comment by dajmo** [comment]
<img src="https://imgur.com/Qhq4NbJ.png" />

**Comment by dajmo** [comment]
<img src="https://imgur.com/vLL6yc4.png" />

**Comment by miklhh** [comment]
Thank you for the screenshot @dajmo, thats a stunning i3 interface you've go going there! :+1:. I love the background.

**Comment by miklhh** [comment]
Could you be more specific with what you mean with network speed. Are you interested in the momentaneous data transfer speed over your network devices?

**Comment by dajmo** [comment]
yes momentaneous data transfer speed , thats what i am requesting

**Comment by uciharis** [comment]
> This i3blocks bar is awesome , I wonder if network-speed meter can be added
> 
> my bar: https://ibb.co/PjSp1R5

the screenshot doesnt exist anymore
